.github/workflows/specimin-evaluation_CI_with_Gradle.yml

```diff
@@ -0,0 +1,66 @@
+name: specimin-evaluation_CI_with_Gradle
+
+on:
+ workflow_dispatch:
+
+jobs:
+  specimin-evaluation:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Set up Java JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '21'  # Replace with the Java version you need (e.g., 11, 16, etc.)
+          distribution: 'adopt'  # Specify the distribution of Java (e.g., adopt, zulu, etc.)
+          architecture: 'x64'  # Specify the architecture of Java (e.g., x64, x86)
+          server-password: ${{ secrets.GITHUB_TOKEN }}  # Use GITHUB_TOKEN for server password
+          overwrite-settings: true  # Overwrite any existing settings
+          check-latest: false  # Do not check for the latest version
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          # pip install -r requirements.txt
+
+      - name: Display CSV Contents
+        run: |
+          echo "Contents of Specimin_Eval_Project_List.csv:"
+          cat Specimin_Eval_Project_List.csv
+
+      - name: Display CSV File Contents
+        run: |
+          cat /home/runner/work/specimin/specimin/Specimin_Eval_Project_List.csv   
+
+      - name: List Files for Debugging
+        run: ls -R
+
+      - name: Clone Project B
+        run: |
+          git clone https://github.com/NiharikaJamble/ASHE_Automated-Software-Hardening-for-Entrypoints ASHE
```

Comment:
Specimin's CI should, if possible, only reference other projects that are under the `njit-jerse` organization. Is it necessary to use your fork, or could you use [this fork](https://github.com/njit-jerse/ASHE_Automated-Software-Hardening-for-Entrypoints) instead?